impeller/renderer/vertex_descriptor.cc

```diff
@@ -17,6 +17,13 @@ bool VertexDescriptor::SetStageInputs(
   for (size_t i = 0; i < count; i++) {
     inputs_.emplace_back(*stage_inputs[i]);
   }
+  // TODO(jonahwilliams): vulkan shader stage needed sorting too. Remove once
+  // autogenerated headers include offset information.
```

Comment:
Please link an issue tracking this work.

Reply:
Done